contents/handbook/engineering/support-hero.md

```diff
@@ -136,6 +136,12 @@ From time to time, customers will request to get their apps added to PostHog Clo
 5. Tell the marketing team about this new integration
 6. Install it on Cloud, and make it global
 
+#### Updating existing apps
+
+1. Open a PR against our forked version of the plugin with the new changes (syncing from the main repo).
+2. Verify the code changes are sane and merge the PR.
```

Comment:
Partial list of things to look out for. I think it would be better to put it as the new `1.` in the `Reviewing new apps` sections above and refer to it, but I cannot comment on unchanged lines :(

```suggestion
2. Review the code changes and merge the PR. Look out for:
  - Proper error handling (plugin emits [RetryError](https://posthog.com/docs/apps/build/reference#maximizing-reliability-with-retryerror) when relevant, instead of throwing unhandled exceptions)
  - Proper use of resources (bounded memory and CPU usage, external requests kept to a minimum)
  - Good security practices (the plugin cannot be used to DDoS some server) 
  - Unit testing coverage when possible
```